### PR TITLE
Login

### PR DESCRIPTION
**Pull request by pgoulding**
Added a loading image and a basic WIP log in component. Not implemented, but there for later if we have time to work on it.

### CODE REVIEW

File: src/containers/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      password:''
+    }
+  }
+
+  login = () => {
+    console.log(this.state)
+  }
+
+  handleChange =(e) => {
+    this.setState={[e.etarget.name]: e.target.value}
```
Comment (KirbyDD) [inline src/containers/Login/index.js]: Does this work? setState is a method that takes in an object for state but you are doing an assignment like it is a variable.

I feel like it should be
this.setState({[e.target.name]: e.target.value})

File: src/containers/ResultsPage/ResultsPage.scss [hunk 2]
```diff
@@ -6,3 +6,17 @@
...
+}
+
+@keyframes roll {
+    0% {
+      transform: rotate(0);
+    }
+    100% {
+      transform: rotate(360deg);
+    }
+}
```
Comment (KirbyDD) [inline src/containers/ResultsPage/ResultsPage.scss]: Them animations woohoo  🤩

File: src/containers/Login/index.js (hunk 1 above)
Comment (pgoulding) [reply src/containers/Login/index.js]: You're totally right, my bad.

Comment (pgoulding) [reply src/containers/Login/index.js]: Fixed Now!

**Pull request merged**